src/main/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfiguration.java

```diff
@@ -48,11 +48,11 @@
 import org.kohsuke.stapler.QueryParameter;
 import org.kohsuke.stapler.StaplerRequest;
 
-import javax.annotation.CheckForNull;
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import edu.umd.cs.findbugs.annotations.CheckForNull;
```

Comment:
Here and below: switching Java annotations to use the Jenkins project's preferred SpotBugs rather than JSR 305, as done in Jenkins core and other plugins in the ecosystem.